source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -164,6 +164,7 @@ public List<PimsProperty> GetAllByIds(List<long> ids)
                 .Include(p => p.Address)
                     .ThenInclude(a => a.Country)
                 .Where(p => ids.Any(s => s == p.PropertyId))
+                .AsNoTracking()
```

Comment:
I checked this, the only consumers are a controller, which just returns the result of the get, and the consolidation logic. This, and the following in the property service was the cause of this issue:

`public List<PimsProperty> GetMultipleById(List<long> ids)
{
    _logger.LogInformation("Getting multiple properties by id");
    _user.ThrowIfNotAuthorized(Permissions.PropertyView);

    List<PimsProperty> properties = _propertyRepository.GetAllByIds(ids);
    foreach (PimsProperty property in properties)
    {
        if (property?.Location != null)
        {
            property.Location = TransformCoordinates(property.Location);
        }
    }

    return properties;
}`